03 - Data Science and Custom Skills/Azure Function Skills/SampleSkills/4_TranslateSkill.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Net.Http;
+using System.Text;
+using System.Collections.Generic;
+
+namespace SampleSkills
+{
+    public static class TranslateSkill
+    {
+        #region Translator Text API Credentials
+        static string path = "https://api.cognitive.microsofttranslator.com/translate?api-version=3.0";
+        // NOTE: Replace this example key with a valid subscription key.
+        static string translatorApiKey = "";
+        #endregion
+
+        #region Class used to deserialize the request
+        public class InputRecord
+        {
+            public class InputRecordData
+            {
+                public string text;
+            }
+
+            public string recordId { get; set; }
+            public InputRecordData data { get; set; }
+        }
+
+        private class WebApiRequest
+        {
+            public List<InputRecord> values { get; set; }
+        }
+        #endregion
+
+        #region Classes used to serialize the response
+        public class OutputRecord
+        {
+            public class ComputerVisionObject
```

Comment:
This shouldn't be here, I see it got copied and pasted from the object detection example.

Reply:
Deleted.